Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3433,6 +3441,144 @@ def get_power_query_results(client: Client, args: dict):
+    )
+
+    # Make request and get raw response
+    installed_applications, pagination = client.list_installed_applications_request(query_params)
+
+    if pagination and pagination.get("nextCursor") is not None:
+        demisto.results("Use the below cursor value to get the next page installed applications \n {}".format(
+            pagination['nextCursor']))
+    all_scopes = []
+    if installed_applications:
+        for each_app in installed_applications:
+            scopes = each_app.get("scopes")
+            if scopes is not None and len(scopes) > 0:
+                for scope in scopes:
+                    scope["applicationCatalogId"] = each_app["applicationCatalogId"]
+                    scope["applicationCatalogName"] = each_app["name"]
+                    all_scopes.append(scope)
+        meta = "Provides summary information and details for all the installed applications that matched your search criteria."
+    else:
+        meta = "The search filters provided are returning no results. Please review and adjust them accordingly."
+
+    context_entries = []
+    for each_scope in all_scopes:
+        entry = {
+            'ID': each_scope.get('id'),
+            'Account': each_scope.get('account'),
+            'AccountId': each_scope.get('accountId'),
+            'ApplicationCatalogId': each_scope.get('applicationCatalogId'),
+            'applicationCatalogName': each_scope.get('applicationCatalogName'),
+            'AlertMessage': each_scope.get('alertMessage'),
+            'CreatedAt': each_scope.get('createdAt'),
+            'Creator': each_scope.get('creator'),
+            'CreatorId': each_scope.get('creatorId'),
+            'DesiredStatus': each_scope.get('desiredStatus'),
+            'HasAlert': each_scope.get('hasAlert'),
+            'LastEntityCreatedAt': each_scope.get('lastEntityCreatedAt'),
+            'Modifier': each_scope.get('modifier'),
+            'ModifierId': each_scope.get('modifierId'),
+            'ScopeId': each_scope.get('scopeId'),
+            'ScopeLevel': each_scope.get('scopeLevel'),
+            'Status': each_scope.get('status'),
+            'UpdatedAt': each_scope.get('updatedAt'),
+            'ApplicationInstanceName': each_scope.get('applicationInstanceName'),
+        }
+        context_entries.append(entry)
+
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            'SentinelOne - List of Installed Applications',
+            context_entries,
+            headerTransform=pascalToSpace,
+            removeNull=True,
+            metadata=meta
+        ),
+        outputs_prefix='SentinelOne.InstalledApps',
+        outputs_key_field='ID',
+        outputs=context_entries,
+        raw_response=installed_applications)
+
+
+def get_service_users_command(client: Client, args: dict) -> CommandResults:
+    """
+    Get all service users matching the input filter
+    """
+    # Get arguments
+    query_params = assign_params(
```

Comment:
we saw in the past if some filters are empty - they are not excepted 
can we please test and make sure we just include filters which exists - drop the keys - if they are empty or not provided - to be safe on runtime

Reply:
Yes, currently we are doing the same, when there is no value exists in the filters then we are dropping them, attaching the screenshots of couple of test scenarios by printing the query params which we sending.
<img width="1364" alt="image" src="https://github.com/user-attachments/assets/ad195f5c-db21-4eff-8de2-d7e961ab9ab0" />
<img width="1364" alt="image" src="https://github.com/user-attachments/assets/0bf5b7d0-5cab-4c72-962a-1bfce18fd8fe" />